Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,63 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+# Tips and Tricks
```

Comment:
No first-order headers, use the Description format instead. Headers are automatically created in all normal viewer programs, including man.serenityos.org